scripts/update/Updater.php

```diff
@@ -140,5 +140,9 @@ public function update($initialVersion)
 
             $this->setVersion('1.5.0');
         }
+
+        if ($this->isVersion('1.5.0')) {
+            $this->setVersion('1.6.0');
+        }
```

Comment:
Shouldn't you use `$this->skipVersion('1.5.0', '1.6.0');` (_I'm not sure about the exact syntax_) ?